nncf/experimental/openvino_native/quantization/quantizer_parameters.py

```diff
@@ -0,0 +1,218 @@
+    :param levels: Number of quantization levels.
+    :param quantizer_config: Config of the quantization configuration.
+    :return: A Tuple
+        level_low - the low quant number
+        level_high - the high quant number
+    """
+    level_high = fix_zero_filters_symmetric(max_values)
+    if is_weights:
+        level_low = -level_high
+    else:
+        signed = quantizer_config.signedness_to_force
+        level_low = np.zeros(level_high.shape) if np.all(min_values >= 0) and not signed else \
+            -level_high * levels / (levels - 2)
+
+    return level_low, level_high
+
+
+def asymmetric_range(min_values: np.ndarray, max_values: np.ndarray,
+                     quantizer_config: QuantizerConfig) -> Tuple[np.ndarray, np.ndarray]:
+    """
+    Calculates the numbers of the low and high quant for the asymmetric quantization scheme.
+
+    :param min_values: Collected min values for the quantized insertion.
+    :param max_values: Collected max values for the quantized insertion.
+    :param quantizer_config: Config of the quantization configuration.
+    :return: A Tuple
+        level_low - the low quant number
+        level_high - the high quant number
+    """
+    level_low, level_high = fix_zero_filters_asymmetric(min_values, max_values)
+    level_low = np.where(level_low < 0.0, level_low, 0.0)
+    level_high = np.where(level_high > 0.0, level_high, 0.0)
+    level_low, level_high = tune_range(level_low, level_high, quantizer_config.num_bits)
+    return level_low, level_high
+
+
+def calculate_weight_quantizer_parameters(weight_tensor: np.ndarray, quantizer_config: QuantizerConfig,
+                                          axis: Optional[int]) -> OVQuantizerLayerParameters:
+    """
+    Calculates FakeQuantize layer attributes for weight quantizer.
+
+    :param weight_tensor: Weight tensor to calculate quantizer attributes.
+    :param quantizer_config: Config of FakeQuantize.
+    :param axis: In per-channel case - the axis for the quantization. In per-tensor - ignored.
+    :return: Parameters of the FakeQuantize layer.
+    """
+    if quantizer_config.per_channel:
+        assert axis is not None
+        axes = list(range(len(weight_tensor.shape)))
+        axes.pop(axis)
+        axes = tuple(axes)
+    else:
+        axes = None
+    min_values = np.amin(weight_tensor, axis=axes, keepdims=True)
+    max_values = np.amax(weight_tensor, axis=axes, keepdims=True)
+
+    levels = compute_levels(quantizer_config, is_weights=True)
+    if quantizer_config.mode == QuantizationMode.SYMMETRIC:
+        level_low, level_high = symmetric_range(min_values, max_values, levels, quantizer_config, is_weights=True)
+    else:
+        level_low, level_high = asymmetric_range(min_values, max_values, quantizer_config)
+
+    output_low, output_high = level_low, level_high
+    return OVQuantizerLayerParameters(min_values, max_values, output_low, output_high, levels)
```

Comment:
Don't we need to broadcast FakeQuantize parameters into the correct shape for per-channel parameters, for example?